### DISCUSSION TITLE
qnap transcoding

### DISCUSSION THREAD

**Discussion by jowtron** [navidrome/navidrome#2662] [Q&A]
After update to 0.50 transcoding no longer works, I'm not sure if or where logs are stored, is it possible to get logs written to the config dir? I'm trying to track down the problem...

**Comment by deluan** [comment]
You have to check with the QNAP package maintainers how to get the logs. Don't know how they have set it up. I'm also using Navidrome on a QNAP, but installed it as a docker container, using Container Station. It is very straightforward and the logs are available in the Container Station's UI or using the `docker` command.
